h2o-genmodel/src/main/java/hex/genmodel/algos/ensemble/StackedEnsembleMojoModel.java

```diff
@@ -27,12 +38,16 @@ public StackedEnsembleMojoModel(String[] columns, String[][] domains, String res
                     basePreds[i * _nclasses + j] = _baseModels[i]._mojoModel.score0(_baseModels[i].remapRow(row), basePredsRow)[j + 1];
                 }
             }
+            if (_metaLearnerTransform.equals("Logit"))
```

Comment:
Since you only have logit transform I think in the reader you could just generate a boolean value to avoid the repeated string comparison.

Reply:
Good idea, thanks!